source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -8612,6 +8687,14 @@ export class AdaptiveCard extends ContainerWithActions {
     get hasVisibleSeparator(): boolean {
         return false;
     }
+
+    public registerMouseEnterCallback(callback: any) {
```

Comment:
Is the goal that all inputs are "revealed" as the mouse enters the Adaptive Card? As opposed to each input being revealed when hovered over?

If so, is that truly what partners want? When I worked with Dynamics on their requirements, they wanted individual inputs to reveal on hover.

Per the screenshots, which match what I thought we were after, that isn't the case. But then I really do not understand all that gymnastic with mouse enter/leave events.